digestparser/parse.py

```diff
@@ -104,17 +107,24 @@ def join_run_tags(run, prev_run, output=''):
     return output
 
 
+def remove_odd_characters(string):
+    """replace invisible whitespace characters"""
+    # LINE SEPARATOR
```

Comment:
what a [generic name](https://www.fileformat.info/info/unicode/char/2028/index.htm) they chose

Reply:
may be worthy of a constant at the top of the file e.g. `LINE_SEPARATOR = "..."`